### ISSUE TITLE
String Support for Adding Custom TLVs

### ISSUE THREAD

**Issue by avlec** [lldpd/lldpd#546]
The LLDP standard specifies alpha-numeric data shall be encoded as UTF-8 in TLVs and my interpretation seemed to indicate that applies to custom-tlvs as well.
A mechanism could be added to `lldpcli` to support entering text that gets ascii encoded. So that the suggested new command format (first command below) would be equivalent to the current way of doing things (second command below).
```
# lldpcli configure lldp custom-tlv oui 00,00,00 subtype 1 oui-info "encoded as utf-8"
# lldpcli configure lldp custom-tlv oui 00,00,00 subtype 1 oui-info 65,6e,63,6f,64,65,64,20,61,73,20,75,74,66,2d,38
```

**Comment by vincentbernat** [comment]
Yes, we could do that. I would suggest to add `oui-info-str` to avoid ambiguity (quoting is done before evaluation of a command, so you cannot know if a string was quoted or not). This is likely doable just by modifying `src/lldpcli/conf-lldp.c`.

**Comment by avlec** [comment]
Yes! I updated my original comment to be a bit more explict/careful with wording.

**Comment by Shal-sudo** [comment]
Hey, I made a change to type oui-info to accept strings. So my local copy of this now supports strings passed in. I can submit it as a patch to this repo. cc: @vincentbernat
